### PR TITLE
Collapse simple template to improve deduplication.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

File: experiment/measurer/test_run_crashes.py [hunk 1]
```diff
@@ -61,3 +61,15 @@ def test_filter_crash_type():
...
+# pylint: disable=protected-access
+def test_filter_crash_state():
+    """Tests _filter_crash_state."""
+    assert (run_crashes._filter_crash_state(
+        'Segv on unknown address:int* std::__1::fill_n<int*, long, int>\n'
+        'int arrow::util::RleDecoder::GetBatchWithDict<int>\n'
+        'parquet::DictDecoderImpl<parquet::PhysicalType<\n') ==
+            'Segv on unknown address:int* std::__1::fill_n\n'
+            'int arrow::util::RleDecoder::GetBatchWithDict\n'
+            'parquet::DictDecoderImpl<parquet::PhysicalType<\n')
```
Comment (mbarbella-chromium) [inline experiment/measurer/test_run_crashes.py:75]: Should we try to handle this case as well? We're in a bit of a weird state either way with the truncated lines, so I don't think too much could go wrong by removing anything after < to the end either.

**Review by mbarbella-chromium — APPROVED** [review]

File: experiment/measurer/run_crashes.py [hunk 2]
```diff
@@ -30,14 +30,21 @@
     'crash_testcase', 'crash_type', 'crash_address', 'crash_state',
     'crash_stacktrace'
 ])
+
 SIZE_REGEX = re.compile(r'\s([0-9]+|{\*})$', re.DOTALL)
+TEMPLATE_REGEX = re.compile(r'<[^>]+>')
```
Comment (jonathanmetzman) [inline experiment/measurer/run_crashes.py:35]: Could you make it explicit that this is C++ templates?
I initial thought this had something to do with HTML templates.

File: experiment/measurer/test_run_crashes.py (hunk 1 above)
Comment (inferno-chromium) [reply experiment/measurer/test_run_crashes.py]: Done, added to regex for truncated stuff.

File: experiment/measurer/run_crashes.py (hunk 2 above)
Comment (inferno-chromium) [reply experiment/measurer/run_crashes.py]: Done!

**Review by jonathanmetzman — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**